src/garage/experiment/local_tf_runner.py

```diff
@@ -44,26 +62,12 @@ class LocalRunner:
 
     """
 
-    def __init__(self, sess=None, max_cpus=1):
-        """Create a new local runner.
-
-        Args:
-            max_cpus(int): The maximum number of parallel sampler workers.
-            sess(tf.Session): An optional tensorflow session.
-                  A new session will be created immediately if not provided.
+    def __init__(self, snapshotter=None, sess=None, max_cpus=1):
+        if snapshotter is not None:
+            self.snapshotter = snapshotter
+        else:
+            self.snapshotter = Snapshotter()
```

Comment:
hmm what if we just don't create any snapshots in this case? where will a snapshotter with no arguments even put its snapshots???

Reply:
Can we assume that, if user doesn't pass `snapshotter` to `LocalRunner`, then they don't want to snapshot at all?

Reply:
i think so. or perhaps we should just have them pass `snapshot_dir` and other optional arguments (mode, etc.), since there is only one Snapshotter class that we can think of.

@krzentner what do you think?